### ISSUE TITLE
<!--!doctype--> is being appended to content pages

### ISSUE THREAD

**Issue by Currey** [skinofthesoul/grav-plugin-automagic-images#6]
Not sure how, or why, but `<!--!doctype-->` is being appended to all content pages once this plugin is installed. It's quite a benign issue, as it is only a comment, but it is an odd issue to have.

**Comment by skinofthesoul** [comment]
True. Pretty sure this also stems from the original plugin which I modified. I plan to work on it soonish.

(Sorry this reply took three years, I got swallowed by a life hole. Back now.)

**Comment by skinofthesoul** [comment]
I’m working on the plugin, and I actually cannot reproduce this. What happens for me is that the first line `<!DOCTYPE html>` from the original Grav output gets **removed**.

Of course it has been a few years now so this might totally be due to changes in Grav, but if you still see this happening, it would be great if you could tell me a little bit more about your setup and how to reproduce this?
